app/_landing_pages/tools.yaml

```diff
@@ -0,0 +1,135 @@
+metadata:
+  title: "Kong Tools"
+  tags:
+    - deck
+    - kic
+    - terraform
+    - konnect-api
+    - admin-api
+content:
+  - row:
+      - column:
+          - type: header
+            config:
+              type: h1
+              text: "Kong Tools"
+              align: center
+
+  - row:
+      - column:
+          - type: structured_text
+            config:
+              header:
+                text: "What are Kong tools?"
+              blocks:
+                - type: text
+                  text: Rate limiting controls the number of API requests a client can make within a specified time frame. 
+                    It's a critical function that helps maintain
+                    the performance and availability of web
+                    services and APIs by controlling the request traffic.
+      - column:
+          - type: mermaid
+            config:
+              alt_text: "A diagram that shows how various Kong tools work, with Konnect as an example"
+              diagram: |
```

Comment:
What's the difference between Konnect and Konnect API?

Reply:
I'm guessing this is referring to the UI, but I'd like clarity on that too.